src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,100 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { request } from '../../api/request';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [film, setFilm] = useState({});
+  const [title, setTitle] = useState('interstellar');
+  const [hasError, findError] = useState(false);
```

Comment:
Как правило используют глагол `set` - установить.
```suggestion
  const [error, setError] = useState(false);
```